plugins/knife/clearwater-dns-records.rb

```diff
@@ -295,6 +295,12 @@ def dns_records
         :value => ipv4s_local_site(find_active_nodes("vellum"), i),
         :ttl   => "60"
       },
+
+      "chronos-site#{i}" => {
+        :type  => "A",
+        :value => ipv4s_local_site(find_active_nodes("vellum"), i),
+        :ttl   => "60"
+      },
```

Comment:
This is to allow Chronos to contact other Chronos processes in different sites (without having to know the exact nodes in the other sites). It goes in chronos_gr.conf (Chronos doesn't use shared config). 

I don't particularly mind if this uses the Chronos or Vellum DNS records; I'll make it consistent though.